lnwallet/channel.go

```diff
@@ -551,6 +551,12 @@ func (lc *LightningChannel) diskHtlcToPayDesc(feeRate chainfee.SatPerKWeight,
 		remoteOutputIndex = htlc.OutputIndex
 	}
 
+	customRecords := htlc.CustomRecords.Copy()
+
+	entryType := entryTypeForHtlc(
+		htlc.CustomRecords, lc.channelState.ChanType,
```

Comment:
I think we should use the copied records here too, so this looks the same everywhere.